lib/engine/g_1856/corporation.rb

```diff
@@ -5,7 +5,7 @@
 module Engine
   module G1856
     class Corporation < Corporation
-      attr_accessor :escrow, :presidents_share
+      attr_accessor :escrow, :presidents_share, :destinations
```

Comment:
we should be moving stuff away from the 1856 specific corporation,

i think other games have destinations as well like 18AL

Reply:
I removed this from the PR altogether. Once I'm able to put together destination checks what `destinations` looks like will be more clear